docs/qa/qa-test-plan.md

```diff
@@ -0,0 +1,332 @@
+<!-- 
+Similar to above - expect same behaviour so perhaps not worth testing gain?
+- [ ] Expect the text before the word to be darker color 
+- [ ] Expect current word to be highlighted
+- [ ] Expect media to start playing and highlight and current word to continue
+ -->
+
+#### Item to test #2.3: Timed Text Editor - Edit speakers labels
+
+#### Steps:
+- click on speaker icon next to default speaker names
+#### Expected Results: 
+-  [ ] Expect a prompt to come up, 
+    - [ ] on add text and click on expect name to change in Timed Text Editor next to speaker label icon clicked.
+- [ ] if instead of ok click cancel, expect nothing to happen
+
+---
+
+### Item to test #3: Settings
+
+#### Item to test #3.1: Settings - show
+
+##### Steps:
+- click on settings, cog  icon top left
+##### Expected Results: 
+-  Expect settings panel to come up 
+
+#### Item to test #3.2: Settings - hide
+
+##### Steps:
+- click on settings panel cross, top right
+##### Expected Results: 
+-  Expect settings panel to hide
+
+
+#### Item to test #3.3: Settings - pause while typing - on
+
+##### Steps:
+- click settings icon top left
+- in settings panel, click toggle for "pause while typing" to be on
+- in TimedTextEditor double click on a word, media starts playing
+- start typing to correct a word
+#### #Expected Results: 
+- [ ] Expect media to pause while typing
+- [ ] Expect media to re-start a few seconds after stopped typing
+
+#### Item to test #3.4: Settings - pause while typing - off
+
+##### Steps:
+- click settings icon top left
+- in settings panel, click toggle for "pause while typing" to be off
+- in TimedTextEditor double click on a word, media starts playing
+- start typing to correct a word
+##### Expected Results: 
+- [ ] Expect media not to pause while typing, and continue playing
+
+#### Item to test #3.5: Settings - scroll sync - on
+scroll into view 
+
+##### Steps:
+- click settings icon top left
+- in settings panel, click toggle for "scroll sync" to be on 
+- in TimedTextEditor click on progress bar, further away from current playhead
+##### Expected Results: 
+- [ ] Expect text in timed text editor in view to jump to corresponding current time
+
+#### Item to test #3.6: Settings - scroll sync - off
+
+##### Steps:
+- click settings icon top left
+- in settings panel, click toggle for "scroll sync" to be off 
+- in TimedTextEditor click on progress bar, further away from current playhead
+##### Expected Results: 
+- [ ] Expect text in timed text editor in view not to jump to corresponding current time
+
+#### Item to test #3.7: Settings - set roll back interval
+
+##### Steps:
+- click settings icon top left
+- in settings panel, change value of 'Rollback Interval (sec)', something other then default 15 seconds
+- Make sure media current time is not zero
+- Click on roll back btn 
+##### Expected Results: 
+-  Expect current time to roll back by the the custom ammount set in previous steps
```

Comment:
Should these also be tickboxes (consistent with above?)

(Same comment for a few other places too)